runtime/src/bank_forks.rs

```diff
@@ -399,6 +399,21 @@ impl BankForks {
         Ok((is_root_bank_squashed, squash_timing))
     }
 
+    /// Returns the interval, in slots, for sending an ABS request
+    fn abs_request_interval(&self) -> Slot {
+        let Some(snapshot_config) = self.snapshot_config.as_ref() else {
+            // If there's no snapshot config, then we shouldn't take snapshots.
+            // Return the max value to prevent sending an ABS request.
+            return Slot::MAX;
```

Comment:
looks like we already have a const for this `DISABLED_SNAPSHOT_ARCHIVE_INTERVAL`. 
can we use that const instead?

Reply:
`Slot::MAX` is just as correct in my opinion. I don't care either way

Reply:
ok. sounds good to me.